partcad/src/partcad/converters/__init__.py

```
@@ -0,0 +1 @@
+from .cad_converter import CADConverter
```

Ruff (0.8.2)

`.cad_converter.CADConverter` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)